applications/tari_app_grpc/src/conversions/sidechain_features.rs

```diff
@@ -435,6 +457,46 @@ impl TryFrom<grpc::CommitteeMembers> for CommitteeMembers {
     }
 }
 
+//---------------------------------- CommitteeSignatures --------------------------------------------//
+impl From<CommitteeSignatures> for grpc::CommitteeSignatures {
+    fn from(value: CommitteeSignatures) -> Self {
+        Self {
+            signatures: value
+                .signatures()
+                .iter()
+                .map(|s| grpc::Signature::from(s.clone()))
```

Comment:
I think this will work
```suggestion
                .into_iter()
                .map(Into::into)
```